dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -161,15 +160,20 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
             GUID taskClsid = details.TaskClsid();
             THROW_HR_IF(E_INVALIDARG, taskClsid == GUID_NULL);
 
-            auto registrationOptions = details.Options();
-            THROW_HR_IF(E_INVALIDARG, WI_AreAllFlagsClear(registrationOptions, PushNotificationRegistrationOptions::PushTrigger | PushNotificationRegistrationOptions::ComActivator));
+            auto registrationActivators = details.Activators();
```

Comment:
NIT: `auto registrationActivators{ details.Activators() };`

[ES.23: Prefer the {}-initializer syntax](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax)

same comment applies throughout